dit-core/src/daemon.rs

```diff
@@ -94,10 +107,14 @@ pub struct LocalListener {
 }
 
 impl LocalListener {
-    pub async fn accept(&mut self) -> io::Result<Option<ConnectionToProcess>> {
+    pub async fn accept(
+        &mut self,
+        controller: Controller,
```

Comment:
Kannst du den `Controller` in dem `LocalListener` speichern, dass man den hier nicht immer übergeben muss?